scripts/lib/setup_venv.py

```diff
@@ -24,10 +24,8 @@
     "libmemcached-dev",
     "python3-dev",          # Needed to install typed-ast dependency of mypy
     "python3-pip",
-    "python-pip",
```

Comment:
This needs to wait for #14278, right?

Reply:
@andersk I think we don't need them for virtualenv (python-pip-whl is enough?). I think they must have been required for some other dependency.

Reply:
Ah, you are correct.